src/NServiceBus.Core/Encryption/Encryptor.cs

```diff
@@ -1,77 +1,17 @@
-﻿namespace NServiceBus.Features
+﻿namespace NServiceBus
```

Comment:
I can easily rename it now it is internal

On Tuesday, August 2, 2016, Tim Bussmann <notifications@github.com> wrote:

> In src/NServiceBus.Core/Encryption/Encryptor.cs
> <https://github.com/Particular/NServiceBus/pull/3991#discussion_r73153013>
> :
>
> > @@ -1,77 +1,17 @@
> > -﻿namespace NServiceBus.Features
> > +﻿namespace NServiceBus
>
> the convention for features would be the features namespace, but I guess
> you changed this because of the obsolete?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3991/files/12adb8c3c3e5ff39ca90e787eb2944f8bbf3c1ef#r73153013>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKosn5Vb8L-mOa7zglqftEQ_GZRF8qfks5qb0RlgaJpZM4JYL5j>
> .
>